pyproject.toml

```
@@ -32,6 +31,12 @@ dependencies = [
     "transformers>=4.56.0",
 ]
 
+[project.optional-dependencies]
+backends = ["vllm>=0.10.0"]
```

How about having one optional dependency group per backend?
```suggestion
huggingface = ["torch", "transformers", "accelerate"]
vllm = ["vllm"]
```

---

sure this makes sense.